eclipse-theia/theia
# fix: return focus to least recently active tab #10533

#### What it does
Fixes #10533: When the last tab of a TabBar is closed, always the last tab of the previous TabBar is activated.
This behavior differs from VSCode - the previously opened tab stays open.

https://user-images.githubusercontent.com/23746778/151578650-a7f0bd0a-004d-47af-a99b-996d0f512e16.mp4

#### How to test
- Open three files
- Move one file into a separate TabBar to the right
- Activate the leftmost tab in the first TabBar, then then single tab in the right TabBar
- Then close the single tab in the right TabBar
- Expected: The least recently activated tab is activated (displayed) in the first TabBar

https://user-images.githubusercontent.com/23746778/151578645-53facb79-cce9-42c7-9710-7609497ab05d.mp4

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

I couldn't spot any existing tests regarding the TabBar behavior - if i have missed them, please let me know. Locally my tests are passing for the core package.
